tests/unittests/BackendTestUtils.h

```diff
@@ -113,6 +113,19 @@ static const auto all_backends = ::testing::Values(
   if (!isEnabledBackend({__VA_ARGS__}))                                        \
     return;
 
+/// Blacklist of tests for the current backend under test.
+extern std::set<std::string> backendTestBlacklist;
+
+#define STR(X) #X
```

Comment:
Could you add doxygen comments to these macros?

Reply:
BTW, it feels like the name `STR` could potentially conflict with something from other libraries. Maybe we should give it a less common name.